README.md

```diff
@@ -2350,27 +2350,24 @@ swaps in its `<body>`, and merges its `<head>`,
 all without incurring the cost of a full page load.
 
 
-Add [turbolinks package](https://www.npmjs.com/package/turbolinks)
-to `/assets/package.json`:
+Luckily, adding `Turbo` will require just a simple
+copy and paste!
+Check the [`unpkg files`](https://unpkg.com/browse/@hotwired/turbo@7.2.4/dist/) 
```

Comment:
is there a _minified_ version of this? 
![image](https://user-images.githubusercontent.com/194400/206768027-01f17ee3-6e26-47fa-9318-fac77b0b400d.png)

![image](https://user-images.githubusercontent.com/194400/206767997-bcf4a86e-f586-46d8-a5bb-60464aa4558e.png)

My quick google yielded no results. 🔍 🤷‍♂️

Reply:
Couldn't find it myself.
https://github.com/hotwired/turbo/issues/18 -> I checked from this issue. It's weird because the CDN in the [Turbo Installation docs](https://turbo.hotwired.dev/handbook/installing) just exports the contents of this file.

Reply:
Yeah, that was my reading too. Leaving it up to the project's packer minify ... seems inefficient. 🐌 
But I get it for source maps 💭